handler/consumer/html/README.md

```diff
@@ -0,0 +1,40 @@
+# Fragment HTML Body Writer
+It is a Fragment Event Consumer that marks the beginning and the end of a fragment with HTML comments:
+```
+<!-- data-knotx-id="FRAGMENT_IDENTIFIER" -->
+  FRAGMENT_BODY
+<!-- data-knotx-id="FRAGMENT_IDENTIFIER" -->
+```
+
+Fragment event is serialized to JSON and stored in `<script>` tag:
+```
+<!-- data-knotx-id="FRAGMENT_IDENTIFIER" -->
+  <script data-knotx-id="FRAGMENT_IDENTIFIER" type="application/json">
+    FRAGMENT_DEBUG_DATA
+  </script>
+  FRAGMENT_BODY
+<!-- data-knotx-id="FRAGMENT_IDENTIFIER" -->
+```
+
+It must be configured in `consumerFactories`
+```hocon
+consumerFactories = [
+  {
+    factory = fragmentHtmlBodyWriter
+    config { CONSUMER_CONFIG }
+  }
+]
+```
+where `CONSUMER_CONFIG` consists of:
+```hocon
+condition {
+  param = debug
+  # header = x-knotx-debug
+}
+fragmentTypes = [ "snippet" ]
+```
+It is triggered when conditions pass. Currently, 2 checks are supported:
+ - `param` - Knot.x expects that the original request contains this request parameter (e.g. by configuring `param=debug`, requesting `{address}?debug=true` will make the condition pass),
+ - `header` condition is analogous, but the value comes from the request header.
```

Comment:
Would be helpful to inform here that the request must contain header with the given _name_ for the header condition to pass (value is irrelevant).
Also, I'd write here that param & header conditions are validated with OR operator between them

Reply:
I agree that it'd be helpful to include the information about only name being important. 
But I think there is no need for specifying that the `or` operator is used. I think it's obvious enough.

Reply:
Combining with:
> It is triggered when conditions pass.

It is not obvious to me, that there is `OR` condition between `param` and `header` checks.

Reply:
> It runs when any of the following conditions are met:

Reply:
> Would be helpful to inform here that the request must contain header with the given _name_ for the header condition to pass (value is irrelevant).
`an original request contains a parameter with the *given name* `